### DISCUSSION TITLE
Scenario "Moyogalpa": Security Snag. The Trials of Mary and John

### DISCUSSION THREAD

**Discussion by Deytron** [SadServers/sadservers#59] [Scenarios]
Interesting scenario with multiple things to fix, but the end solution doesn't work for me? apparmor_parser command is only usable with root (which makes sense), but even after reloading the added lines to the profile, the access is still Forbidden. 

Did I miss something?

**Comment by fduran** [comment]
Hard to say, it's possible the AppArmor config is not quite right.

The clues that were meant early on as simple instructions are not a good way to show for example changes in a config file (eg indenting) or complex solutions. This is my bad and in my TODO I have a task to show clues or full solution in a proper document like the ones at https://docs.sadservers.com/docs/ but somehow protected.

**Reply by Mileshin** [reply]
Hi @Deytron 
In addition to modifying AppArmor, you also need to adjust the permissions for the /home/webapp/static-files directory. The current permissions set to 640 are not appropriate.
